src/test/java/BaseTest.java

```diff
@@ -6,13 +6,19 @@
 import org.openqa.selenium.safari.SafariDriver;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.BeforeSuite;
+import org.openqa.selenium.WebElement;
+
+import java.time.Duration;
 
 
 public class BaseTest {
 
     @BeforeSuite
-    static void setupClass() {
-        WebDriverManager.chromedriver().setup();
+     static void setupClass() {
+
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
```

Comment:
Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/225541863-88379921-cf12-430a-91e2-da299e05541a.png)

Hint

1.
 After the Chrome 111 update, you can no longer kick off a chromedriver instance unless you add an additional chrome option:
"--remote-allow-origins=*"

```
        ChromeOptions options = new ChromeOptions();
        options.addArguments("--disable-notifications","--remote-allow-origins=*", "--incognito","--start-maximized");
        driver = new ChromeDriver(options);
```
2.

@BeforeMethod allows the method to execute before the execution of each @Test methods, whereas @afterMethod is executed after the execution of each @Test methods.

![image](https://user-images.githubusercontent.com/104761482/225546413-d3dd0290-d04b-4acd-9c08-13a6f64b0f61.png)

3.
![image](https://user-images.githubusercontent.com/104761482/225546508-cba6f4b5-e736-4a98-adef-7252e29b0dea.png)

4.
![image](https://user-images.githubusercontent.com/104761482/225546606-33867325-94a7-49d2-b53f-c3e080a9db70.png)

5.

![image](https://user-images.githubusercontent.com/104761482/225546708-063bbabe-2cac-42e8-9df9-b49fb080bc53.png)

Please, check the login and password once again.

![image](https://user-images.githubusercontent.com/104761482/225546794-61c04ee0-900a-4f6d-837b-edf57c6976c5.png)